### PR TITLE
CATROID-516 Remove redundant if statement

### PR DESCRIPTION
**Pull request by adnaribo**
Removing redundant if statements. https://jira.catrob.at/browse/CATROID-516

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/ev3/EV3Reply.java [hunk 1]
```diff
@@ -35,10 +35,7 @@
 
 	@Override
 	public boolean hasError() {
-		if (getStatusByte() == NO_ERROR) {
-			return false;
-		}
-		return true;
+		return getStatusByte() == NO_ERROR;
```
Comment (dandrawes) [inline catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/ev3/EV3Reply.java:38]: This should be: `return getStatusByte() != NO_ERROR;`

File: catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/nxt/NXTReply.java [hunk 2]
```diff
@@ -45,10 +45,7 @@
 
 	@Override
 	public boolean hasError() {
-		if (getStatusByte() == NO_ERROR) {
-			return false;
-		}
-		return true;
+		return getStatusByte() == NO_ERROR;
```
Comment (dandrawes) [inline catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/nxt/NXTReply.java:48]: This should be: `return getStatusByte() != NO_ERROR;`

**Review by dandrawes — CHANGES_REQUESTED** [review]
Please fix the two small stated issues.

**Review by dandrawes — APPROVED** [review]
Looks good now!

**Comment by wslany** [comment]
Thanks!

**Pull request merged**